package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "metal-myComponent",
+	"version": "0.0.0",
+	"description": "My awesome Metal.js project",
+	"license": "BSD",
+	"repository": "zxdgoal/metal-myComponent",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"main": "src/MyComponent.js",
+	"files": [
+		"src",
+		"test"
+	],
+	"scripts": {
+		"test": "gulp test"
+	},
+	"keywords": [
+		"metal"
+	],
+	"dependencies": {
+    "bootstrap": "^3.3.6",
+		"closure-templates": "20140422.0.0",
+		"metal": "metal/metal.js#master",
```

Comment:
All the metal dependencies should point to rc versions instead of master. Take a look at [metal-modal](https://github.com/metal/metal-modal/blob/master/package.json) for example (this is useful for the other things I've mentioned to be changed in this file too, metal-modal's package.json is following the right pattern.